golem_messages/datastructures/tasks.py

```diff
@@ -120,7 +120,7 @@ def sign_task(self, private_key: bytes) -> None:
     def verify(self, public_key: bytes) -> None:
```

Comment:
please change the return type:
```suggestion
    def verify(self, public_key: bytes) -> None:
```
```suggestion
    def verify(self, public_key: bytes) -> bool:
```

Reply:
I disagree. The only interesting effect of this function is whether it raises or not.